server/db.js

```diff
@@ -14,85 +14,94 @@ const config = {
   dropDatabase: true,
 };
 
-// schema of each individual review
-const reviewSchema = new mongoose.Schema({
-  id: mongoose.ObjectId,
-  user_icon: String,
-  review_text: String,
-  date: Date,
-  name: String,
-  rating: {
-    cleanliness: Number,
-    communication: Number,
-    check_in: Number,
-    accuracy: Number,
-    location: Number,
-    value: Number
-  }
-});
-
-db.on('error', console.error.bind(console, 'connection error:'));
-db.once('open', function() {
-  // populate the database once we are connected
-  initDB();
-});
-
 // get random number between min and max, inclusive
 const getRandomInt = (min, max) => {
   let range = max - min;
   return Math.floor(Math.random() * range) + min;
 }
 
-const random_names = ['Chris', 'Katie', 'Emmanuel', 'Josef', 'Kiara', 'Karen', 'Clarence', 'Jorge', 'Antonio',
+const randomNames = ['Chris', 'Katie', 'Emmanuel', 'Josef', 'Kiara', 'Karen', 'Clarence', 'Jorge', 'Antonio',
                       'Elana', 'Lim', 'Jake', 'James', 'Johnny', 'Jorgen', 'Haneen', 'Mataeux', 'Theo', 'Ryan',
                       'Jacob', 'Jenny', 'Alex', 'Alissa', 'Andrew', 'Anna', 'Arun', 'Adjoa', 'Billy', 'Brian',
                       'Carina', 'Catherine', 'Daniel', 'Erfan', 'Eric', 'Harris', 'Harrison', 'Jen', 'Jessie',
                       'Joel', 'Johnny', 'Joesph', 'Karl', 'Katharine', 'Liz', 'Mike', 'Minji', 'Mylani', 'Rebecca',
                       'Rob', 'Shaquon', 'Sophie', 'Sokhary', 'Susan', 'Victoria', 'Watson', 'Yas'];
 
-const random_text = ['apple', 'orange', 'pear', 'little', 'big', 'hack', 'reactor', 'review', 'great', 'the', 'a',
-                     'is', 'of', 'will', 'tree', 'normal', 'abstract', 'your', 'job', 'free', 'work', 'stay','home'.
+const randomText = ['apple', 'orange', 'pear', 'little', 'big', 'hack', 'reactor', 'review', 'great', 'the', 'a',
+                     'is', 'of', 'will', 'tree', 'normal', 'abstract', 'your', 'job', 'free', 'work', 'stay','home',
                      'apartment', 'kite', 'rent', 'stay', 'like', 'roof', 'room', 'bathroom', 'lorem ipsum'];
 
 const generateRandomReview = (length) => {
```

Comment:
Dont need to pass length in as an arg, as you have already written a getRandomInt function that can be used inside of this function to generate a random length of a review.

Reply:
https://github.com/keybox130/Reviews-Service/pull/1/files/0b9b2ebdee1f09bca326960678f7fc69a9b5ba3c..014a120f3f4739003f7d084f60d9fe05b115aaec#r495200218